force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -15494,12 +15494,4 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
         <shortDescription>Label to define aria-label on donation history datatable</shortDescription>
         <value>Donation History</value>
     </labels>
-    <labels>
```

Comment:
There's a reference to this label on [Line 9 of donationHistoryTable.js](https://github.com/SalesforceFoundation/NPSP/pull/6885/files#diff-82afee8e56b0856526a516547fb461c5fab8a66bd54f62c991cb5e06aa018cd8R9) that is causing a build error.

Reply:
Thank you, Ben. It has now been removed.